### DISCUSSION TITLE
[Bug] Can't find sp_key

### DISCUSSION THREAD

**Discussion by yoohooyoo** [KRTirtho/spotube#426] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Can't find “sp_key”, the new version of the spotify field has been renamed to “sp_phash”.
It is recommended that this step get integrated to spotube. Integrate a login page and intercept the corresponding field.

### Expected Behavior

It is recommended that this step get integrated to spotube. Integrate a login page and intercept the corresponding field.

### Operating System

Windows

### Operating System Version

win10

### Installation Source

Github Release (Binary)

**Comment by KRTirtho** [comment]
Can you specify your region?

**Comment by mohd-murshid** [comment]
couldn't find `sp-key` or `sp_phash`. Is there any other options i need to check

**Comment by KRTirtho** [comment]
@t4technow You've to first log in at [spotify](https://accounts.spotify.com/en/login) :joy:

**Comment by cd-FileX** [comment, marked as answer]
Now it is `sp_gaid`

**Reply by KRTirtho** [reply]
@yoohooyoo if above works for you mark this as solution

**Comment by forteller** [comment]
It was sp_key for me when I did it right now.

**Comment by rvbsm** [comment]
Can't login because of that maybe (after clicking Submit button nothing happens). For me Spotify hasn't `sp_key` nor `sp_phash`, but there is `sp_gaid`. My Region: UA

**Comment by ghost** [comment]
Also for me there is no `sp_key`.

These are all spotify's `sp_` cookies:

![image](https://github.com/KRTirtho/spotube/assets/118444583/a1176fc0-9e66-41b3-81f3-dc784fa4a90b)

I used `sp_gaid` in place of `sp_key` and it seems to work. spotube logged in.

**Reply by KRTirtho** [reply]
If it works, then I'll mention it in the tutorial as well
Does it work for everyone facing the similar issue?

**Reply by ovizii** [reply]
I'm sorry, I am new to this app, so I am wondering what tutorial you are talking about? Would you mind sharing a link please?

**Reply by KRTirtho** [reply]
It's the sign in "Step by step tutorial" in the app login page (only in desktop)
